modules/persistent-storage-csi-sc-managing-cli.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc
+//
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-sc-managing-cli_{context}"]
+= Managing the default storage class using the CLI
+
+.Prerequisites
+* Administrator rights.
```

Comment:
Is this sufficient? I see multiple instances of: _You have access to the cluster as a user with the `cluster-admin` role._.

Reply:
Fixed